functions/api/v1/join-us/submit.js

```diff
@@ -0,0 +1,74 @@
+import jwt from '@tsndr/cloudflare-worker-jwt';
+
+export async function onRequest(context) {
+	if (context.request.method !== 'POST') {
+		return new Response('Method not allowed', { status: 405 })
+	}
+
+	const body = await context.request.json();
+	const { name, email, country, token } = body;
+
+	if (!name || !email || !country || !token) {
+		return new Response('Missing required fields', { status: 400 });
```

Comment:
I don't know how much extra work, but it may be reasonable to indicate missing fields?

Reply:
You should never run into this, this is a private API